python/tvm/script/ir_builder/relax/ir.py

```diff
@@ -125,7 +125,7 @@ def tensor(
 
 Object = ObjectType()  # pylint: disable=invalid-name
 Shape = ShapeType()  # pylint: disable=invalid-name
-Void = TupleType(None)  # pylint: disable=invalid-name
+Void = TupleType([])  # pylint: disable=invalid-name
```

Comment:
This intention is to make it consistent with https://github.com/mlc-ai/relax/blob/f5dd68b0ef2a70a8db220877e65fe1cab1a68570/include/tvm/ir/type.h#L377